lib/kafka.ts

```diff
@@ -58,3 +58,75 @@ export const waitForKafkaReady = async function (page: Page, name: string) {
     timeout: config.kafkaInstanceCreationTimeout
   });
 };
+
+export const manageAccess = async function (page: Page, id: string, desc: string) {
+  await page.getByTestId('actionManagePermissions').click();
+
+  await page.getByRole('button', { name: 'Options menu' }).click();
+
+  await page.getByRole('option', { name: id + ' ' + desc }).click();
+
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  await page.getByTestId('permissions-dropdown-toggle').click();
+
+  await page
+    .getByRole('menuitem', {
+      name: 'Produce to a topic Provides access to produce to one or more topics depending on topic selection criteria'
+    })
+    .click();
+
+  await page.getByPlaceholder('Enter prefix').click();
+
+  await page.getByPlaceholder('Enter prefix').fill('test');
+
+  await page.getByTestId('permissions-dropdown-toggle').click();
```

Comment:
Same here.

Reply:
By selecting the Tab by text `await page.getByText('Access').click();` it finds another button not desired:

![image](https://user-images.githubusercontent.com/100594859/205652603-be111493-fd7d-4fd3-9be6-67160a943759.png)

Reply:
Can I resolve this conversation? @mmusil